docs/scos/dev/guidelines/nfr-guidelines.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Project Development Non-Functional Requirement guidelines
+description: @TODO
+last_updated: @TODO
+template: concept-topic-template
+related:
+- title: Operatable feature guidelines
+  link: docs/scos/dev/guidelines/operatable-feature-guidelines.html
+- title: Process documentation guidelines
+  link: docs/scos/dev/guidelines/process-documentation-guidelines.html
+- title: Monitorable process guidelines
+  link: docs/scos/dev/guidelines/monitorable-process-guidelines.html
+---
+
+
+## Introduction
+
+Non-Functional Requirements (NFRs) are one of the core tools for architects and developers to describe how the system under development supposed to work from technical perspective.
+Defining NFRs can be as important as defining functional requirements, which are usually done by business people (i.e. Product Owners). Missing out on them usually result in
+too late realization of certain unintended behavior that costs a lot of resources/money for the projects to deal with.
+
+## NFR guidelines
+
+To ensure that the most important aspects are not missed to be considered when defining NFRs, we suggest using the Software Quality Attributes like Availability, Security, Performance, etc.
+You can use the [ISO 25010 standard](https://iso25000.com/index.php/en/iso-25000-standards/iso-25010) as a baseline for Quality Attributes, but you can also come up your own Quality Attributes (QAs)
```

Comment:
As a baseline (that you offer below as example), what do you think of referencing the [PASSME](https://nick-goupinets.medium.com/passme-muster-846a9997645b) here? @helen-laktionova Please suggest - do we rely on Passme in our SA training materials?